### PR TITLE
feat(pivotgrid): add PivotGrid for Material theme

### PR DESCRIPTION
**Pull request by PreslavKozovski**

### CODE REVIEW

File: packages/material/scss/pivotgrid/_theme.scss [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+    .k-pivot-toolbar .k-button,
+    .k-fieldselector .k-list li.k-item {
+        @include appearance( button );
+        background-clip: padding-box;
+        background-color: $pivotgrid-button-bg;
+        box-shadow: unset;
+
+        .k-field-actions{
+            top: initial;
```
Comment (elena-gancheva) [inline packages/material/scss/pivotgrid/_theme.scss]: Shouldn't this be in `_layout` file?

File: packages/material/scss/pivotgrid/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    }
+
+    .k-pivot-toolbar .k-button,
+    .k-fieldselector .k-list li.k-item {
+        @include appearance( button );
+        background-clip: padding-box;
+        background-color: $pivotgrid-button-bg;
+        box-shadow: unset;
+
+        .k-field-actions{
```
Comment (elena-gancheva) [inline packages/material/scss/pivotgrid/_theme.scss]: Keep attention to whitespaces - we have whitespace after selector and curly bracket

File: packages/material/scss/pivotgrid/_theme.scss [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+        .k-i-loading::before,
+        .k-i-loading::after {
+            background-color: $header-bg;
+        }
+    }
+
+    .k-fieldselector,
+    .k-pivot{
+        .k-i-arrow-60-right::before, .k-i-arrow-e::before, .k-i-sarrow-e::before, .k-i-expand::before, .k-i-expand-e::before {
```
Comment (elena-gancheva) [inline packages/material/scss/pivotgrid/_theme.scss]: You can use nesting here
```css
.a, .b, .c {
    &::before { //logic }
}
```

File: packages/default/scss/pivotgrid/_layout.scss [hunk 4]
```diff
@@ -84,6 +83,10 @@
         border-left-style: solid;
     }
 
+    .k-pivot-table .k-grid-header-wrap {
+        margin-right: -1px;
```
Comment (elena-gancheva) [inline packages/default/scss/pivotgrid/_layout.scss]: Why do we need this negative margin?

Comment (PreslavKozovski) [reply packages/default/scss/pivotgrid/_layout.scss]: The above line fixes this issue:
https://www.screencast.com/t/jjyyhnj00hCD

If there is a better approach to achieve the desired behavior, please, let me know :)

**Pull request merged**